src/pages/home/Employee/EmployeeHome.js

```diff
@@ -89,7 +89,7 @@ function EmployeeHome() {
                 marginLeft: "auto"
               }}
          >
-           <Avatar imageLink={user?.picture} />
+           <Avatar imageLink={user?.picture} logout={logout} email={user?.email} name={user?.name} />
```

Comment:
We can rename the Avatar component as AvatarWithPopper

Reply:
Done